product_operating_unit/models/product_template.py

```diff
@@ -18,18 +15,25 @@ def _default_operating_unit_ids(self):
                      self.env.uid).id])]
 
     operating_unit_ids = fields.Many2many(
-        'operating.unit', 'product_operating_unit_rel',
-        string='Operating Unit',
-        default=_default_operating_unit_ids
-    )
+        'operating.unit',
+        'product_operating_unit_rel',
+        string='Operating Units',
+        default=_default_operating_unit_ids)
```

Comment:
With this new approach, the default ones shouldn't be the ones from the category instead of those from the user?
Apart from that, for existing products, I guess we need a migration script that initializes them.